crates/bevy_gizmos/src/gizmos.rs

```diff
@@ -13,6 +13,7 @@ type PositionItem = [f32; 3];
 type ColorItem = [f32; 4];
 
 const DEFAULT_CIRCLE_SEGMENTS: usize = 32;
+const DEFAULT_ARC_SEGMENTS: usize = 32;
```

Comment:
Given that arcs are shorter than circles, is this the correct default? I would expect this to be 16 personally.

Reply:
An arc could be anywhere from 10% to 90% of a circle. 32 makes the worst case(the arc forms a circle) have the same detail as a circle. Which seems reasonable.

The amount of segments adjusting to the length of the arc might be cool. I wouldn't block on that though.

Reply:
We could lerp the number of segments with the length of the arc, and only if we use the default value. What do you think?

Reply:
Eh, just leave it :)

Reply:
oh well I went ahead and did the linear interpolation nonetheless :)